websites/M/Mavanimes/presence.ts

```diff
@@ -0,0 +1,86 @@
+const mavanimes = new Presence({
+    clientId: "814986239681626143"
+  });
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
```

Comment:
use <Presence>.getTimestamps() instead

Reply:
I don't know how to use Presence#getTimestamps()

Reply:
```suggestion
```